modules/core/src/main/java/org/apache/ignite/internal/processors/metric/impl/MetricUtils.java

```diff
@@ -33,7 +33,10 @@
  */
 public class MetricUtils {
     /** Metric name part separator. */
-    public static final String SEPARATOR = ".";
+    public static final char SEPARATOR_CHAR = '.';
+
+    /** Metric name part separator. */
+    public static final String SEPARATOR = "" + SEPARATOR_CHAR;
```

Comment:
JS style detected %) You don't use `SEPARATOR_CHAR` anywhere except of `SEPARATOR` value. 

It is enough:

public static final String SEPARATOR = ".";

Or use `SEPARATOR_CHAR` in `indexOf` and `lastIndexOf`. 

Any way `"."` looks better than `"" + SEPARATOR_CHAR`, IMHO